datapower-docker-compose-dp-mq/datapower-docker-compose-dp-mq.mkd

```diff
@@ -0,0 +1,151 @@
+## A DataPower and MQ environment with Docker Compose 
+This sample demonstrates how to utilize multiple containers, orchestrated by Docker Compose, to create an environment that includes DataPower receiving and processing a request and writing the resulting message to an MQ back end. Subsequent support containers are included to send requests and store style sheets and schema utilized by the DataPower processing policy. In this sample, I'll outline the travels of the request message, `bob`, through the pipeline created in the docker-compose environment.
```

Comment:
- [ ] Rework `Subsequent support` sentence to something like `The composed application includes everything you need -- the load driver, the DataPower, and the MQ back-end. Everything is there from config to stylesheets to docker-compose.yml`
- [ ] why `pipeline`? It is a term-of-art in devops, it is a term-of-art for MQ?
- [ ] docker-compose isn't an environment